src/main/java/com/amazonaws/services/sqs/AmazonSQSRequesterClientBuilder.java

```diff
@@ -15,6 +15,7 @@
     private Map<String, String> queueAttributes = Collections.emptyMap();
     
     private int idleQueueSweepingPeriod = 5;
+    private long idleQueueHeartbeatInterval = 5;
```

Comment:
Here and throughout, I'd take out the "idle" part and just go with `queueHeartbeatInterval`, since the whole point is that queues with heartbeats aren't idle!

Reply:
Removed the "idle" part of the variable and methods